apps/showcase/src/app/modules/use-http-image-source-pipe-showcase/use-http-image-source-pipe-showcase.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { UseHttpImageSourcePipeModule } from '@this-dot/ng-utils';
+
+import { UseHttpImageSourcePipeShowcaseRoutingModule } from './use-http-image-source-pipe-showcase-routing.module';
+import { UseHttpImageSourcePipeShowcaseComponent } from './components/use-http-image-source-pipe-showcase/use-http-image-source-pipe-showcase.component';
```

Comment:
Maybe it would be better to rename this to eg. `NgUtilsShowcaseRoutingModule` and `NgUtilsShowcaseComponent`? But we can do this later as well

Reply:
I created #52 to address this issue. For now I'll leave it as it is.